packages/client/README.md

```diff
@@ -11,7 +11,7 @@
 [![codecov](https://img.shields.io/codecov/c/github/ipfs/helia-routing-v1-http-api.svg?style=flat-square)](https://codecov.io/gh/ipfs/helia-routing-v1-http-api)
 [![CI](https://img.shields.io/github/actions/workflow/status/ipfs/helia-routing-v1-http-api/js-test-and-release.yml?branch=main\&style=flat-square)](https://github.com/ipfs/helia-routing-v1-http-api/actions/workflows/js-test-and-release.yml?query=branch%3Amain)
 
-> A Routing V1 HTTP API client
+> A [Delegated Routing V1](https://specs.ipfs.tech/routing/http-routing-v1/) HTTP API client
```

Comment:
When `aegir check-project` is run this is pulled from the `description` field in `package.json` which doesn't support markdown, please revert.